src/fsharp/FSharp.Core/list.fs

```diff
@@ -223,7 +223,7 @@ namespace Microsoft.FSharp.Collections
 
         [<CompiledName("Pairwise")>]
         let pairwise (list: 'T list) =
-            Microsoft.FSharp.Primitives.Basics.List.pairwise list
```

Comment:
@liboz  Looking back over this commit, I'd recommend we don't make this change.  The qualified name was used for a reason - to show that this function is defined in the ``Basics.List`` module.  Otherwise it looks like the function is self-referential, and that's confusing.